### ISSUE TITLE
Question: Unable to autocomplete field.

### ISSUE THREAD

**Issue by jorgecc** [klesun/deep-assoc-completion#111]
I checked some issues but I am still unable to autocomplete a field.

```php
class Example {
    /** @var string=['a','b','c'][$i] */
    var $field;
}

$obj=new Example();

$obj->field=''; // it fails to autocomplete
```
* I don't need $obj->field['...']
* The phpdoc string=['a','b','c'][$i]  works with arguments of a function.

**Comment by klesun** [comment]
So, if I get it right, you want plugin to suggest the string value in an assignment. Thanks for the suggestion. I guess I'll add it to my todo list.

**Comment by klesun** [comment]
The feature is now available in *testing* release [2019.09.18.002](https://plugins.jetbrains.com/files/9927/68956/deep-assoc-completion.jar?updateId=68956&pluginId=9927&family=INTELLIJ)

Will deploy to stable in few days

**Comment by klesun** [comment]
In stable now
